README.md

```diff
@@ -169,6 +174,60 @@ It's always worth noting that you should generate, vet, format, lint, and test y
 make test
 ```
 
+## CLI
+
+A CLI tool is available to make it easy to deploy new Kogito Services from source instead of relying on CRs yaml files.
+
+### CLI Requirements
+
+1. [`oc` client](https://docs.okd.io/latest/cli_reference/get_started_cli.html) installed
+2. An authenticated OpenShift user with permissions to create resources in a given namespace
+3. [Go installed](https://golang.org/doc/install) and available in your `PATH`
+4. Kogito Operator [installed](#installation) in the cluster
+
+### Build CLI from source
+
+Build the CLI by running:
+
+```bash
+$ git clone https://github.com/kiegroup/kogito-cloud-operator
+$ cd kogito-cloud-operator
+$ make install-cli
+```
+
+The `kogito` CLI should be available in your path:
+
+```bash
+$ kogito
+Kogito CLI deploys your Kogito Services into an OpenShift cluster
+
+Usage:
+  kogito [command]
+
+Available Commands:
+  app         Sets the Kogito application where your Kogito Service will be deployed
+  deploy      Deploys a new Kogito Service into the application context
+  help        Help about any command
+  new-app     Creates a new Kogito Application for your Kogito Services
+
+Flags:
+      --config string   config file (default is $HOME/.kogito.json)
+  -h, --help            help for kogito
+  -v, --verbose         verbose output
+
+Use "kogito [command] --help" for more information about a command.
+```
+
+After [installing](#installation) the Kogito Operator, it's possible to deploy a new Kogito Service by using the CLI:
+
+```bash
+#creates a new namespace/kogito app in your cluster
+$ kogito new-app kogito-cli
+
```

Comment:
@ricardozanini I suppose that here should be a step to install Kogito Operator into newly created namespace.
What about adding capability to install available Kogito Operator into the newly created namespace, in case it is missing?

Reply:
Sure, it's in the plans, but we don't have bandwidth to do it for 0.3. Do you mind opening a JIRA to track this? I'll try to follow up on it as soon as I get the operator managed by the OperatorHub.io. Installing manually via CLI will be prune for errors.

Reply:
Ok, reported as https://issues.jboss.org/browse/KOGITO-169 .